fence/config-default.yaml

```diff
@@ -751,6 +751,7 @@ SEND_TO: 'example@gmail.com'
 # NOTE: Example in comments below
 AWS_SES:
   SENDER: ""
+  SENDER_NAME: ""
```

Comment:
What is SENDER_NAME used for here?

Reply:
That is unnecessary and should not be there.  Removing.